docs/elements.md

```diff
@@ -721,6 +723,52 @@ def generate(data):
+| `allow-blank`              | boolean                                      | false        | Whether or not an empty input box is allowed. By default, empty input boxes will not be graded (invalid format).                                                                                                                                                                        |
+| `blank-value`              | string                                       | ""           | Value to be used as an answer if element is left blank. Only applied if `allow-blank` is true. Must follow the same format as an expected user input.                                                                                                                                   |
+| `size`                     | integer                                      | 35           | Size of the input box.                                                                                                                                                                                                                                                                  |
+| `show-help-text`           | boolean                                      | true         | Show the question mark at the end of the input displaying required input parameters.                                                                                                                                                                                                    |
+| `placeholder`              | string                                       | -            | String to override default placeholder text. The default placeholder gives information about the comparison type used.                                                                                                                                                                  |
+| `magnitude-partial-credit` | float                                        | -            | Fraction of partial credit given to answers of correct magnitude and incorrect units when `grading-mode=exact-units`. Remaining fraction of credit given when units are correct but magnitude is incorrect. Must be between 0.0 and 1.0. Partial credit is disabled if this is not set. |
+| `show-feedback`            | boolean                                      | true         | Whether to show feedback for incorrect answers.                                                                                                                                                                                                                                         |
```

Comment:
What is this controlling exactly? If this option has the exact same effect of using `<pl-hide-in-panel>`, we should remove this option and just have people use that.

Reply:
Whether or not detailed feedback is shown (so like telling people whether their specific unit or magnitude is off). This doesn't do the same thing as hide in panel, as that will hide the correct answer along with the feedback, and instructors should have the granularity.

Reply:
Sounds good, maybe add a little bit more of that info to the description here so its more clear.

Reply:
Added some discussion in https://github.com/PrairieLearn/PrairieLearn/pull/6790/commits/4d5b245479e0260867e69994f65d9d5b8dd71cdb. Could add more discussion but the feedback given is actually pretty rudimentary.